packages/e2e-tests/utils/account.js

```diff
@@ -4,10 +4,14 @@ const E2eTestAccount = require("./E2eTestAccount");
 const getBankAccount = async () => {
     const { BANK_ACCOUNT: accountId, BANK_SEED_PHRASE: seedPhrase } = process.env;
     const account = new E2eTestAccount(accountId, seedPhrase, { accountId: nearApiJsConnection.config.networkId });
-    await account.initialize();
-    return account;
+    return account.initialize();
 };
 
+function generateTestAccountId() {
+    return `test-playwright-account-${Date.now()}-${Math.floor(Math.random() * 1000) % 1000}`;
```

Comment:
Did we decide to abbreviate this? :D

Reply:
Done here: https://github.com/near/near-wallet/pull/2174/files#diff-c2c58188c2514e87ba540c8f2d7d767e40fe9f361f3f6bae1ab31e68f75c56cbR20